bigquery/CHANGELOG.md

```diff
@@ -4,6 +4,29 @@
 
 [1]: https://pypi.org/project/google-cloud-bigquery/#history
 
+## 0.32.0
+
+- BigQuery: Moves all Table property conversion logic into properties (#5125)
```

Comment:
- Moves all Table ...
- Moves all Dataset ...
- Remove unnecessary _Table ...
- Removes _ApiResourceProperty ...
- BigQuery: Add dev version for 0.32.0 release (#5105)
- shorten snippets test name (#5091)
- Bigquery: remove test_load_table_from_uri_w_autodetect_schema_then_get_job because of duplicate test in snippets (#5004)

These changes were purely refactoring changes, which shouldn't have had any public interface changes. Should they appear in the changelog?

Reply:
new version out. I stripped these lines. I don't know the philosophy of the changelog/release notes but these seem fine to drop. New PR inbound.